### DISCUSSION TITLE
Is nuxt3 support planned?

### DISCUSSION THREAD

**Discussion by mendrinos** [ivodolenc/nuxt-animejs#9] [Q&A] [answered]
Hello,

I was wondering if there are plans to support Nuxtjs 3?

Thanks!

**Comment by ivodolenc** [comment]
Hi, 

I think Nuxt 3 is still in beta, so let's wait for the first official production release. Just to note, the module works fine in Nuxt 2 projects. 

For now, you can manually import Anime.js to your Nuxt 3 project.

**Comment by ivodolenc** [comment, marked as answer]
Hi, I finally updated the module so feel free to try the new version.

The module is optimized and supports nuxt 3 with typescript. Have fun!
